projects/ngx-dropzone/src/lib/ngx-dropzone.service.spec.ts

```diff
@@ -2,8 +2,8 @@ import { TestBed, inject } from '@angular/core/testing';
 import { NgxDropzoneService } from './ngx-dropzone.service';
 
 // Mockup FileList class for unit tests
-class FileList {
-  private files: File[] = [];
+class MockFileList implements FileList {
```

Comment:
I got the following typescript errors for each call to `parseFileList` with the previous version:

´´`
TS2345: Argument of type 'FileList' is not assignable to parameter of type 'FileList'.   Index signature is missing in type 'FileList'.
```

The service seems to expect a `FileList` interface from `lib.dom.d.ts`.